cfpb/cfgov-refresh
# Ask CFPB misc front end additions

Miscellaneous front end modifications for Ask pages

## Additions

- `Round` option for `category-icon` on Ask category pages
- `flush-left` list modifier for cards on Ask landing page
- Button text customization for `pagination`

## Testing

- Check out branch
- Set up Ask if necessary
  ```
  export DEPLOY_ENVIRONMENT=build
  ./drop-db.sh
  ./create-mysql-db.sh
  mysql -uroot v1 < ~/Downloads/production_django.sql
  python cfgov/manage.py migrate
  python cfgov/manage.py runscript migrate_knowledgebase
  ```
- Verify that bulleted lists in category cards on [Ask landing page](http://localhost:8000/ask-cfpb/) are aligned flush left
- Verify round category icons and `Next`/`Previous` pagination button text on [Ask category pages](http://localhost:8000/ask-cfpb/category-auto-loans/)

## Screenshots

### Flush left list modifier
<img width="1171" alt="screen shot 2017-05-30 at 7 03 09 pm" src="https://cloud.githubusercontent.com/assets/778171/26612757/96a02ad2-456b-11e7-8fbc-d15675c52583.png">

### Round category icons
<img width="1186" alt="screen shot 2017-05-30 at 7 03 27 pm" src="https://cloud.githubusercontent.com/assets/778171/26612762/9c4283c2-456b-11e7-8771-85b6cc84d490.png">

### Custom pagination button text
<img width="930" alt="screen shot 2017-05-30 at 7 03 42 pm" src="https://cloud.githubusercontent.com/assets/778171/26612766/a0a0a7b4-456b-11e7-96c8-400c350098b4.png">


## Checklist
